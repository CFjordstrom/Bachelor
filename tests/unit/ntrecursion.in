{
#1 -> b #2 | d #4;
#2 -> c #3;
#3 -> a #1;
#4 -> #5 #5;
#5 -> 123;
}
#1