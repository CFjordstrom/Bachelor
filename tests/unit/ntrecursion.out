-mindfa
{
#1 -> d #2 | b #3;
#2 -> 1 #4;
#3 -> c #5;
#4 -> 2 #6;
#5 -> a #1;
#6 -> 3 #7;
#7 -> 1 #8;
#8 -> 2 #9;
#9 -> 3 #10;
#10 -> ();
}
#1